packages/config/config/devices/0x0344/he-ht01.json

```diff
@@ -0,0 +1,795 @@
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"20": {
+			"label": "Time Format",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "24 hour",
+					"value": 0
+				},
+				{
+					"label": "12 hour (AM / PM)",
+					"value": 1
+				}
+			]
+		},
+		"21": {
+			"label": "Week Day Override",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Monday",
+					"value": 1
+				},
+				{
+					"label": "Tuesday",
+					"value": 2
+				},
+				{
+					"label": "Wednesday",
+					"value": 3
+				},
+				{
+					"label": "Thursday",
+					"value": 4
+				},
+				{
+					"label": "Friday",
+					"value": 5
+				},
+				{
+					"label": "Saturday",
+					"value": 6
+				},
+				{
+					"label": "Sunday",
+					"value": 7
+				}
+			]
+		},
+		"22": {
+			"label": "Time Override",
+			"unit": "HHMM",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2359,
+			"defaultValue": 0
+		},
+		"23": {
+			"label": "Time Regulation Reg On Time",
+			"unit": "minutes",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 240,
+			"defaultValue": 30
+		},
+		"24": {
+			"label": "Time Regulation Reg Off Time",
+			"unit": "minutes",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 240,
+			"defaultValue": 30
+		},
+		"25": {
+			"label": "Dry Time",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 90,
+			"defaultValue": 30
+		},
+		"26": {
+			"label": "Mode To Switch After Dry Mode Operation Completion",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "COM",
+					"value": 1
+				},
+				{
+					"label": "TIME",
+					"value": 2
+				},
+				{
+					"label": "ECO",
+					"value": 4
+				},
+				{
+					"label": "VAC",
+					"value": 5
+				},
+				{
+					"label": "MAN",
+					"value": 6
+				}
+			]
+		},
+		"41": {
+			"label": "Morning Start Time",
+			"unit": "HHMM",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2359,
+			"defaultValue": 600
+		},
+		"42": {
+			"label": "Day Start Time",
+			"unit": "HHMM",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2359,
+			"defaultValue": 900
+		},
+		"43": {
+			"label": "Evening Start Time",
+			"unit": "HHMM",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2359,
+			"defaultValue": 1800
+		},
+		"44": {
+			"label": "Night Start Time",
+			"unit": "HHMM",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 2359,
+			"defaultValue": 2300
+		},
+		"45": {
+			"label": "Monday Morning Temp",
+			"description": "Allowable range: 1 - 37, x10",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 370,
+			"defaultValue": 240
+		},
```

Comment:
See comments above, please apply them here (and the following parameters) too.

Reply:
removed description